vm/src/value.rs

```diff
@@ -247,11 +247,11 @@ impl Value {
         }
     }
 
-    pub fn get_float<ENV>(&self, _vm: &GVm<ENV>) -> VMResult<f64> {
+    pub fn get_float<ENV>(&self, _vm: &GVm<ENV>) -> VMResult<f32> {
```

Comment:
Did you mean to drop this back to f32?  Doesn't this cause a lot of code to not even use the extra precision in f56?